### PR TITLE
1.3.12 Changelog

### PR DESCRIPTION
**Pull request by smackesey**
## Summary & Motivation

Changelog.

### CODE REVIEW

**Comment by smackesey** [comment]
Current dependencies on/for this PR:
* master
  * **PR #15010** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/15010" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/15010?utm_source=stack-comment).

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,57 @@
...
 
+# 1.3.12 (core) / 0.19.12 (libraries)
+
+### New
+
+- The `--name` argument is now optional when running `dagster project from-example`
+- An asset key can now be directly specified via the asset decorator: `@asset(key=...)`.
+- `AssetKey` now has a `with_prefix` method.
+- Significant performance improvements when using `AutoMaterializePolicy`s with large numbers of partitions.
+- `dagster instance migrate` now prints information about changes to the instance database schema
```
Comment (jfineberg) [inline CHANGES.md:11]:
```suggestion
- `dagster instance migrate` now prints information about changes to the instance database schema.
```

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,57 @@
...
+- The [`dagster-cloud-agent` helm chart](https://artifacthub.io/packages/helm/dagster-cloud/dagster-cloud-agent) now supports setting K8s labels on the agent deployment.
+- [ui] Step compute logs are shown under “Last Materialization” in the asset sidebar.
+- [ui] Truncated asset names now show a tooltip when hovered in the asset graph.
+- [ui] The “Propagate changes” button has been removed and replaced with “Materialize Stale and Missing” (which was the “Propagate changes” predecessor)
```
Comment (jfineberg) [inline CHANGES.md:15]:
```suggestion
- [ui] The “Propagate changes” button has been removed and replaced with “Materialize Stale and Missing” (which was the “Propagate changes” predecessor).
```

File: CHANGES.md [hunk 3]
```diff
@@ -1,5 +1,57 @@
 # Changelog
 
+# 1.3.12 (core) / 0.19.12 (libraries)
+
+### New
+
+- The `--name` argument is now optional when running `dagster project from-example`
```
Comment (jfineberg) [inline CHANGES.md:7]:
```suggestion
- The `--name` argument is now optional when running `dagster project from-example`.
```

**Review by jfineberg — APPROVED** [review]

**Pull request merged**